src/main/java/com/github/onsdigital/JWTKeyProviderImpl.java

```diff
@@ -13,6 +13,7 @@
  * JWTKeyProviderImpl - Fetches JWT signing keys from a service according to the configs passed to it.
  */
 public class JWTKeyProviderImpl implements JWTKeyProvider {
+    private static final String JWT_KEYS_ROUTE = "/v1/jwt-keys";
```

Comment:
Is there a reasonably easy way we can provide this value instead of hardcoding it? I think it will be more flexible and resistant to future changes. As it is now we would have to release a new version of the lib if we needed to update the URI, it would nicer/simpler if we could just update the config of the app using it.